install_deps.sh

```diff
@@ -6,7 +6,7 @@ set -x -e
 
 if [ "${ID_LIKE}" = debian -o "${ID}" = debian ]
 then
-    sudo add-apt-repository ppa:graphics-drivers/ppa
+    sudo add-apt-repository-repository ppa:graphics-drivers/ppa
```

Comment:
"repository" is duplicated?

Reply:
Resolved the `apt-install-repository` problem (strange thing).

Maybe @Hyjale has insight on why this is necessary?

Reply:
This was added right before the ADA release and was necessary before installing `glslang-tools`.